go/enclave/rollupchain/rollup_chain.go

```diff
@@ -674,18 +674,33 @@ func (rc *RollupChain) GetBalance(encryptedParams common.EncryptedParamsGetBalan
 		return nil, fmt.Errorf("could not decrypt params in eth_getBalance request. Cause: %w", err)
 	}
 
+	var paramList []string
+	err = json.Unmarshal(paramBytes, &paramList)
+	if err != nil {
+		return nil, fmt.Errorf("failed to unmarshal RPC request params from JSON. Cause: %w", err)
+	}
+	if len(paramList) < 2 {
+		return nil, fmt.Errorf("required exactly two params, but received zero")
+	}
+	address := gethcommon.HexToAddress(paramList[0])
```

Comment:
what happens if this fails?

Reply:
It can't fail. It'll take your string, convert it to bytes, and stuff up to 20 of them into the address.

Not the best API, but we use this everywhere. Maybe we need our own `SafeHexToAddress` or similar?

Reply:
yeah. can you add a todo?

Reply:
Done